File: core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/downscale/DownscaleClusterFailurePayload.java
package com.sequenceiq.cloudbreak.core.flow2.cluster.downscale;

import com.sequenceiq.cloudbreak.core.flow2.cluster.ClusterScaleFailedPayload;

public class DownscaleClusterFailurePayload extends ClusterScaleFailedPayload {

    public DownscaleClusterFailurePayload(Long stackId, String hostGroupName, Exception errorDetails) {
        super(stackId, hostGroupName, errorDetails);
    }
}

File: core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/downscale/DownscaleClusterFailurePayload.kt
package com.sequenceiq.cloudbreak.core.flow2.cluster.downscale

import com.sequenceiq.cloudbreak.core.flow2.cluster.ClusterScaleFailedPayload

class DownscaleClusterFailurePayload(stackId: Long?, hostGroupName: String, errorDetails: Exception) : ClusterScaleFailedPayload(stackId, hostGroupName, errorDetails)
